lib/galaxy/webapps/galaxy/api/tool_shed_repositories.py

```diff
@@ -704,6 +704,45 @@ def repair_repository_revision(self, trans, payload, **kwd):
         # Display the list of repaired repositories.
         return tool_shed_repositories
 
+    @expose_api
+    @web.require_admin
+    def purge_repository(self, trans, id=None, **kwd):
+        """
+        Purge a "white ghost" repository from the database.
```

Comment:
I know that's what we have in the equivalent controller method, but "white ghost" is not great as description.
```suggestion
        Purge a repository that is stuck in "NEW" state from the database.
```

Reply:
+1